chromium_src/ios/web/web_state/ui/crw_web_controller.mm

```diff
@@ -9,9 +9,14 @@
 
 #include <Webkit/Webkit.h>
 
+#define _navigationHandler                                         \
+  _navigationHandler =                                             \
```

Comment:
`if (false)` so we don't actually initialize CRWWKNavigationHandler and throw it away, although maybe this gets optimized out?

Reply:
Even if it does get initialized it'll just dealloc at the end of scope and it doesn't do anything in the initializer. I could add the `if(false)` check though if we want to be future proofed

Reply:
I understand it will be dealloced, but we shouldn't create it in the first place since we don't need it

Reply:
The `if(false)` check was added in the last commit, forgot to mention it https://github.com/brave/brave-core/pull/26957/commits/6ce599224697cb68893f07642955023dec9cfb06